.github/workflows/samples.yml

```diff
@@ -44,14 +48,13 @@ jobs:
           path: ./sample
           persist-credentials: false
 
-      - name: Set imports.deno-slack-api/ to ../deno-slack-api/src/ in import_map.json
+      - name: Set imports.deno-slack-api/ to ../deno-slack-api/src/ in imports
```

Comment:
🔍 The logging in this updated file is so helpful!

https://github.com/slackapi/deno-slack-api/actions/runs/15397083256/job/43320475967#step:5:205

```
`import file` out content:
...
    "deno-slack-api/": "../deno-slack-api/src/",
```